### PR TITLE
Change Torch hub Directory and Add Citation

### PR DESCRIPTION
**Pull request by Luckygyana**

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# YAML 1.2
+---
+abstract: "We describes the steps necessary to port OpenL3 audio embeddings1(Cramer et al., 2019)to PyTorch2.  We do regression tests to demonstate and elucidate differences between themodels.  The largest mean absolute error (<1e−3) occurs when using mel-spectrogramson high-pitched frequency.Keywords:Audio Embedding, pytorch, Mel Spectrogram, Spectrogram"
+authors: 
+  -
+    family-names: "Gyanendra Das"
```
Comment (turian) [inline CITATION.cff:6]: Family name is last name

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# YAML 1.2
+---
+abstract: "We describes the steps necessary to port OpenL3 audio embeddings1(Cramer et al., 2019)to PyTorch2.  We do regression tests to demonstate and elucidate differences between themodels.  The largest mean absolute error (<1e−3) occurs when using mel-spectrogramson high-pitched frequency.Keywords:Audio Embedding, pytorch, Mel Spectrogram, Spectrogram"
```
Comment (turian) [inline CITATION.cff:3]: Can you make this on several lines? Also, typos. 

No one calls things PyTorch2. Please use the name and capitalization that is standard

File: setup.py [hunk 3]
```diff
@@ -39,8 +37,8 @@
         "resampy>=0.2.1,<0.3.0",
         #'scikit-image>=0.14.3,<0.15.0',
         "torch>=1.4.0",
-        "nnAudio",
-        "julius",
+        "nnAudio>=0.2.4",
+        "julius>=0.2.5",
```
Comment (turian) [inline setup.py:41]: Does it work if you don't pin the version

Comment (Luckygyana) [reply setup.py:41]: No. It doesn't. I checked

File: CITATION.cff (hunk 2 above)
Comment (Luckygyana) [reply CITATION.cff:3]: Okay

File: CITATION.cff (hunk 1 above)
Comment (Luckygyana) [reply CITATION.cff:6]: Oh, I will change

**Review by turian — APPROVED** [review]

**Pull request merged**